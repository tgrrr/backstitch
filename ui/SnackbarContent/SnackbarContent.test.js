import React from 'react'
import renderer from 'react-test-renderer'
import { SnackbarContent } from '.'

it('SnackbarContent: default', () => {
    const component = renderer.create(
        <SnackbarContent>
            <>
                SnackbarContent
            </>
        </SnackbarContent>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
