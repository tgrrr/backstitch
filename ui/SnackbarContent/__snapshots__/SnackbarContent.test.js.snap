// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnackbarContent: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Snackbarcontent "
    data-testid="Snackbarcontent"
  >
    <div
      className="MuiPaper-root MuiSnackbarContent-root MuiPaper-elevation6"
      role="alert"
    >
      <div
        className="MuiSnackbarContent-message"
      />
    </div>
  </div>
</div>
`;
